client-libraries/rust/modality-mutator-protocol/Cargo.toml

```diff
@@ -4,7 +4,7 @@ version = "0.1.0"
 edition = "2021"
 license = "Apache-2.0"
 repository = "https://github.com/auxoncorp/modality-sdk"
-description = "The protocol definition of the Modality mutation plane."
+description = "The protocol definition of the Modality mutation plane mutator."
```

Comment:
Is this change a meaningful distinction that I just don't understand?

Reply:
It could be improved, I'm open to alternative suggestions here just needed them to be different for the time being.
The `modality-mutation-plane` crate is mostly concerned with the mutation plane protocol itself, whereas the `modality-mutator-protocol` crate deals more with constituent mutators.